css-align-3/self-alignment/place-self-shorthand-001.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<title>CSS Box Alignment: place-self shorthand - single values specified</title>
+<link rel="author" title="Javier Fernandez" href="mailto:jfernandez@igalia.com" />
+<link rel="help" href="http://www.w3.org/TR/css3-align/#place-self-property" />
+<meta name="assert" content="Check that setting a single value to place-self expands to such value set in both 'align-self' and  'justify-self'." />
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/css-align-3/resources/alignment-parsing-utils.js"></script>
+<div id="log"></div>
+<script>
+    var values = ["normal", "stretch"].concat(selfPositionValues, baselineValues);
```

Comment:
This is a comment unrelated to this tests, but shouldn't have we added `stretch` for `place-items` too?